KVMarketSquare/Scenes/HomeView.swift

```diff
@@ -46,6 +53,14 @@ struct HomeView: View {
         }
         .padding(.horizontal, 32)
     }
+    
+    private var shopsListView: some View {
+        HStack {
```

Comment:
This can be `VStack` with `ScrollView` OR we can have `List` here if we want to have vertically scrolled list

Reply:
Was thinking of a wrapper `HStack` but either works. I can make those changes in the next PR.

Reply:
Yeah no worries now. I think we will work on this again anyways. 👍